docs/contributing/database-migrations/edd.mdx

```diff
@@ -173,49 +216,82 @@ END
 </TabItem>
 </Tabs>
 
-## Workflow
+## Our EDD Process
+
+There are some unique constraints to Bitwarden that are not addressed directly in Martin Fowler's
+EDD article.
+
+- Our Production instances in the cloud are required to be on at all times
+- Our self-host instances that we do not directly have access to manage must support the same EDD
+  processes; however, they do not have the same always-on application constraint
+- Minimization of manual steps in our process
```

Comment:
I've updated the language a bit to focus more on the implementation of orchestrating EDD in our environment rather than EDD itself.

After reading through Fowler's article, I came away with the impression that the purpose of EDD is not to support an always-on DB/application. He went so far as to say that it would take a whole separate article to examine the technicalities to orchestrate EDD in such an environment. I think it is important to keep the always-on requirement for orchestration in mind.

I see this a different way. Exiting the Transition Phase is not done until the Finalization migrations have been run in the next update. If every release of server has destructive DB changes, I would say that the self-host deployment will constantly switch from one Transition Phase to the next.